session1/solutions/assignment3/Rakefile

```diff
@@ -1 +1,20 @@
 # assignment 3 solution
+
+require 'fileutils'
+namespace :rename_yml do
```

Comment:
This is wrong. namespace was supposed to be :files and the actual task :rename_yml

Reply:
I changed it to the correct one
![2018-01-10_12-05-18](https://user-images.githubusercontent.com/12675378/34766924-0f123e8a-f5ff-11e7-922f-b12acedb5cd1.jpg)